data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+	on complete
+		event "remnant: postverta reveal"
+		conversation
+			`You arrive at the hidden station on Esquiline, where the Gavriil is waiting to offload the atmosphere. His team quickly hooks up the gascraft up to ports on the bay walls and the equipment starts disgorging its contents into the system. After a few minutes they disconnect the ships, and you and the gascrafts head up to the pelican to refill your reservoirs and ferry it down to the station.`
+			`	As the last load starts to empty into the station another tech comes up to you. "Captain, we just received a message from Prefect Chilia. He would like you to meet him on Viminal as soon as possible." You nod and start to run checks on the <ship>, making ready to leave once the cargo is done unloading.` 
+
+
+
+event "remnant: postverta reveal"
+	system Postverta
+		asteroids "large metal" 9 3
+		asteroids "large rock" 4 3
+		asteroids "medium metal" 52 1
+		asteroids "medium rock" 4 2
+		government Uninhabited
+		haze _menu/haze-red
+		link Diespiter
+		link Prosa
+		link Statina
+		object "Ssil Vida"
+			distance 1408
+			period 554
+			sprite planet/station_emitter1
+		object
+			distance 1208
+			period 654
+			sprite planet/rock10
+		object
+			distance 439
+			period 143
+			sprite planet/desert7
+		object
+			distance 814
+			period 362
+			sprite planet/fog0
+			object
+				distance 202
+				period 21
+				sprite planet/dust4
+		object
+			period 19
+			sprite star/m4
+		pos 297 269
+	unvisit Levana
+	unvisit Caeculus
+	unvisit Prosa
+	unvisit Egeria
+	system Levana
+		add object wormhole_barren
+			distance 2313
+			period 389
+			sprite planet/wormhole-red
+	system Caeculus
+		add object wormhole_barren2
+			distance 2400
+			period 391
+			sprite planet/wormhole-red
+	system Prosa
+		add object wormhole_barren2
+			distance 2313
+			period 389
+			sprite planet/wormhole-red
+	system Egeria
+		add object wormhole_barren
+			distance 2313
+			period 389
+			sprite planet/wormhole-red
+		
+
+
+mission "Remnant: Cognizance 12"
+	name "Investigate strange sensor readings"
+	description "Long range sensors have picked up strange gravitational waves. Go investigate."
+	landing
+	source "Viminal"
+	waypoint
+		system Postverta
+	stopover
+		planet "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 11: done"
+	on offer
+		conversation
+			`You stop by the starport on and are quickly accosted by a Remnant who informs you that Prefect Chilia is on his personal warship, and escorts you to a nearby starling. Inside you find Chilia and several other Remnant clustered around a holographic display and some display panels. "Ah, Captain. Join us, please." Chilia gestures for you to join them around the displays. As you approach, you note that the holographic display shows the Ember Waste, although there are a few details that do not match normal star maps.`
```

Comment:
`You land by the starport and is immediately accosted by a Remnant who informs you that Prefect Chilia is on his personal warship before escorting you to a nearby starling. Inside you find Chilia and several other Remnants clustered around a holographic display and some display panels. "Ah, Captain. Join us, please." Chilia gestures for you to join them around the displays. As you approach, you note that the holographic display shows the Ember Waste, although there are a few details that do not match normal star maps.`

Reply:
Fixed. Thanks!